setup.cfg

```diff
@@ -3,7 +3,7 @@
 # http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
 
 [metadata]
-name = core
```

Comment:
@brianthelion  these chages were neccaary for the current test suitte to pass. This should during @quincy2112 refactor to make this repo namespace conformant

Reply:
Can't parse ^^^. You're saying that updating to `f0cal.core` now will break the test suite, so just updating to `f0cal` is an intermediate step? If so, then ok, proceed.